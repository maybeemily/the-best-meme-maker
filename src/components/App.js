import React from 'react';
import Meme from '../containers/Meme';
import Header from './Header';
import Footer from './Footer';

export default function App() {
  return (
  <>
  <Header/>
  <Meme />
  <Footer />
  </>
  );
}
