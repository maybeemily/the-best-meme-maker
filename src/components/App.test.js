import React from 'react';
import App from './App';
import { shallow } from 'enzyme';

describe('App rendering test', () => {
  it('renders the app', () => {
    const wrapper = shallow(<App/>);
    expect(wrapper).toMatchSnapshot();
  });
});
