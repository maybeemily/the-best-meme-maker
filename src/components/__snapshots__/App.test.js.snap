// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App rendering test renders the app 1`] = `
<Fragment>
  <Header />
  <Meme />
  <Footer />
</Fragment>
`;
